<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(36, 31, 49);</string>
  </property>
  <widget class="QLabel" name="plaintext_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(98, 160, 234);</string>
   </property>
   <property name="text">
    <string>Plaintext:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="encrypt_button">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>430</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 191, 188);</string>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
  <widget class="QLabel" name="shift_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(98, 160, 234);</string>
   </property>
   <property name="text">
    <string>Shift:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>430</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 191, 188);</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decrypt_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>430</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 191, 188);</string>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
  </widget>
  <widget class="QLabel" name="ciphertext_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(98, 160, 234);</string>
   </property>
   <property name="text">
    <string>Ciphertext:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ciphertext_field">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plaintext_field">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);</string>
   </property>
  </widget>
  <widget class="QLabel" name="cipher_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>801</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tibetan Machine Uni</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(28, 113, 216);</string>
   </property>
   <property name="text">
    <string>Caesar cipher</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="shift_spin_box">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>315</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(98, 160, 234);</string>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>25</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="crack_button">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>324</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sawasdee</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);</string>
   </property>
   <property name="text">
    <string>Crack</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
